test/unit/query/builder.js

```diff
@@ -2,21 +2,31 @@
 /*eslint no-var:0, indent:0, max-len:0 */
 'use strict';
 
-var MySQL_Client = require('../../../lib/dialects/mysql')
-var PG_Client = require('../../../lib/dialects/postgres')
-var Oracle_Client = require('../../../lib/dialects/oracle')
-var Oracledb_Client = require('../../../lib/dialects/oracledb')
-var SQLite3_Client = require('../../../lib/dialects/sqlite3')
-var MSSQL_Client = require('../../../lib/dialects/mssql')
+var _typeof2 = require('babel-runtime/helpers/typeof');
+
+var _typeof3 = _interopRequireDefault(_typeof2);
+
+var _keys = require('babel-runtime/core-js/object/keys');
```

Comment:
I turned the lambdas to regular callback functions using babel. I didn't notice the styling and other side-effects it caused but was just happy that I don't need to do it by hand and unit tests were successful on each platform.
Anyhow, are you happy at all with the way how I refactored the verification logic in the query builder unit test? I wouldn't like to spend additional hours on this and then hearing you disprefer it for any reasons.